starter-code/public/javascripts/financial-data.js

```diff
@@ -0,0 +1,68 @@
+axios.get('http://api.coindesk.com/v1/bpi/historical/close.json')
```

Comment:
Keep your base URL in a variable at the top of your file and concatenate params if need.  This way the base url will be the same for every call.  At the moment you use `http:` here and `https:` in your other request.